### ISSUE TITLE
Confused

### ISSUE THREAD

**Issue by akubryant** [danielkrupinski/VAC-Bypass#33]
What's different beetwen VAC-Bypass and VAC-Bypass-Loader?

**Comment by danielkrupinski** [comment]
Loader loads (injects a .dll) bypass for you.

**Comment by MrClue** [comment]
Wait so if i just use the loader, do i still need to compile this project for a DLL? @danielkrupinski

**Comment by ghost** [comment]
No, it is included in the loader. Please close this issue as it is old and unnecessary if you actually read the other repo.
